unlock-app/src/components/content/MembersContent.tsx

```diff
@@ -165,6 +166,24 @@ const MetadataTableWrapper = ({
     currentPage
   )
 
+  const [filtredItems, setFiltredItems] = useState<any[]>([])
+
+  useEffect(() => {
+    setFiltredItems(list)
+  }, [])
+
+  const search = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const search = e?.target?.value ?? ''
+    const searchTermEmpty = search.length === 0
+
+    if (searchTermEmpty) {
+      setFiltredItems(list)
+    } else {
+      const filtredResults = searchFromList(list, search, columns)
+      setFiltredItems(filtredResults)
+    }
+  }
+
```

Comment:
Instead of storing filtered items. 

I would only store the query and do search on each render pass using fuse.

```ts
const [query, setQuery] = useState('')
const fuseSearch = useMemo(() => { return new Fuse(items, {})}, [items])
const items = fuse.search(query) // any fallbacks can be configured in options or via ternary here
// render
return items.map(item => (<div> {item.value} </div> ))
```